proto/cosmos/tx/v1beta1/service.proto

```diff
@@ -18,7 +19,10 @@ service Service {
   rpc GetTx(GetTxRequest) returns (GetTxResponse) {
     option (google.api.http).get = "/cosmos/tx/v1beta1/tx/{hash}";
```

Comment:
not related to this PR, but I also noticed this, we could also:
```diff
- "/cosmos/tx/v1beta1/tx/{hash}"
+ "/cosmos/tx/v1beta1/txs/{hash}"
```

this matches what we do in modules, e.g. `"/cosmos/staking/v1beta1/validators/{validator_addr}"`, wdyt?

Reply:
ACK.

Reply:
Done in 7c3ad41317b5f9a61d37b79f1ea9a25a4a7b754a